client/analysisui/formatdef.js

```diff
@@ -89,6 +102,21 @@ FormatDef.variables = new Format({
         return r;
     },
 
+    toAriaLabel: function(raw) {
+        if (raw.length === 1)
+            return s_(`Variable {0}`, raw);
```

Comment:
is this for the results table? if so, i think we need to read very carefully interpreting a cell value as a variable. ... we can reliably infer that a value is an interaction by the presence of the asterisk ... but inferring that a value is a variable is more challenging.

i think this will provide wrong inferences at times ... inferring that something is a variable when it isn't (perhaps in other modules), or not realising that something is a variable when it is ...

i'd like to roll out a mechanism that let's us do this reliably ... and just report `supp` rather than `Variable supp` for now.

Reply:
No this is not for the tables. This is specifically for things that have been specified as type variable. If it just text it refers to it as supp. This will never provide any wrong inferences.